Create academia-hub.md (#1611)
huggingface/hub-docs: docs/hub/academia-hub.md

# Academia Hub

<Tip>
<a href="https://huggingface.co/contact/sales?from=academia" target="_blank">Get in touch</a> from your university-affiliated email address to initiate the subscription process.
</Tip>

> Academia Hub provides every student, reseacher, or faculty member having a verified and currently active email address from your academic institution access to our Hub’s PRO features. Explore advanced tools, enhanced collaboration, and exclusive resources to accelerate your learning and research.

Reviewer: ```suggestion
Academia Hub provides the [Hub’s PRO features](https://huggingface.co/pricing#pro) to every student, researcher, or faculty member of an academic institution. Explore advanced tools, enhanced collaboration, and exclusive resources to accelerate your learning, research, and teaching.
```

I simplified by removing the eligibility details because they're below anyway.

<a href="https://huggingface.co/contact/sales?from=academia" class="flex justify-center">
    <img class="block" src="https://huggingface.co/datasets/julien-c/documentation-images/resolve/main/AcademiaHub.png" />
</a>

#### How does it work

Any affiliated user will need to add their `@your-university-name.edu` email address to their HF account.

Academia Hub is designed for:
- **Students:** Unlock powerful features to learn about AI and Machine learning in the most efficient way.
- **Researchers:** Collaborate with peers using the standard AI ecosystem of tools.
- **Faculty members:** Enhance your classes' projects with PRO capabilities.

Key Features are:

- **Full PRO Access:** Enjoy all premium functionalities on the Hub.
- **Seamless Onboarding:** Quick and secure sign-up with your academic email.
- **Enhanced Collaboration:** Connect with peers and mentors.
- **Exclusive Resources:** Access datasets, models, and projects tailored for academia.

#### Eligibility

- Must possess a valid university or college email address.
- Open to all students regardless of discipline or level of study.

> - Pricing: Academia Hub is priced based on volume of usage and number of active users. 

Reviewer 2: Can they self-subscribe or is it in a per-case basis?

Author: i think it's through a contact with our team only

- **Spaces Hosting:** Create ZeroGPU Spaces with A100 hardware.
- **Spaces Dev Mode:** Fast iterations via SSH/VS Code for Spaces.
- **HF Inference API:** Get x20 higher rate limits on HF Serverless API.
- **Dataset Viewer:** Activate it on private datasets.
- **Blog Articles:** Publish articles to the Hugging Face blog.
- **Social Posts:** Share short updates with the community.
- **Inference Providers:** Get $2 included credits across all Inference Providers.
- **Seamless Onboarding:** Quick and secure sign-up with your academic email.
- **Enhanced Collaboration:** Connect with peers and mentors.
- **Exclusive Resources:** Access datasets, models, and projects tailored for academia.

#### Eligibility

- Must possess a valid university or college email address.
- Open to all students regardless of discipline or level of study.
- Pricing: Academia Hub is priced based on volume of usage and number of active users. 
#### Subscribe

<a href="https://huggingface.co/contact/sales?from=academia" target="_blank">Get in touch</a> or send an email to academia@hf.co to initiate the subscription process to the Academia Hub today.

#### Private storage

> Academia Hub includes 1TB of [private repository storage](./storage-limits) per seat in the subscription, i.e. if your institution has 400 students and faculty, then you have 400TB included storage for their private models and datasets.

Author: idk 😆 i was counting on you to tell me

> Academia Hub includes 1TB of [private repository storage](./storage-limits) per seat in the subscription, i.e. if your institution has 400 students and faculty, then you have 400TB included storage for their private models and datasets.

Reviewer 3: I envisioned it more as an org - that way some students can utilise more storage whilst some others less and they can then use the storage across personal projects etc AND the uni org - not sure how feasible this is tho.

> Academia Hub includes 1TB of [private repository storage](./storage-limits) per seat in the subscription, i.e. if your institution has 400 students and faculty, then you have 400TB included storage for their private models and datasets.

Reviewer 4: Based on our current state of the art, doing it through orgs is the simplest (they would just have to enable autojoin based on email domain I think for this to work?)

> they can then use the storage across personal projects etc AND the uni org

^that part is a big no though 😄 - that's why I was wondering about doing it in personal namespaces instead (it would be relatively easy to grant a free PRO upon validation of an email address associated with the University)

Alternatively, we could do it in both - 1TB in the personal namespace, and 1TB/seat in the org 💥

> Academia Hub includes 1TB of [private repository storage](./storage-limits) per seat in the subscription, i.e. if your institution has 400 students and faculty, then you have 400TB included storage for their private models and datasets.

Author: > it would be relatively easy to grant a free PRO upon validation of an email address associated with the University

Yep i think it's the way to go as a prototype

> Academia Hub includes 1TB of [private repository storage](./storage-limits) per seat in the subscription, i.e. if your institution has 400 students and faculty, then you have 400TB included storage for their private models and datasets.

Author: more precisely:
- email address associated with the University gets you to auto-join the university's main org (which is mostly an umbrella org with read-access, i.e. no content there)
- being inside the org gets your user account the equivalent of PRO
